### DISCUSSION TITLE
[FR] default slug value

### DISCUSSION THREAD

**Discussion by spenya** [ComfortablyCoding/strapi-plugin-slugify#62] [Ideas]
I would like to give a suggested slug, but that user were able to change it at their own, so it would work only if slug field is empty ...

**Comment by ComfortablyCoding** [comment]
A default slug option per content type is something I am ok with. Athough I dont see the use case for a hardcoded default, do you mind explaining why this is needed?

**Comment by spenya** [comment]
for exemple, we create a new content, with a title, the suggested slug would be, the title slugified, but the user would be able to change it to something more SEO optimized, maybe because they want the SEO focus on something else different than the title ...
